doc: add activity page with explanations on monitoring re-deployments (#347)
CleverCloud/documentation: content/doc/administrate/activity.md

---
type: docs
title: Activity
description: Applications activity lifecycle explained
tags:
- lifecycle
keywords:
- activity
- unreachable
- timeout
- diskfull
- scaling
- maintenance
type: docs
---

> Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application restarts, a new block appears on the timeline. Here's what you might see on the timeline:

Reviewer: It can be a bit confusing.
Redeploy can be a restart or a rebuild
Dunno if this distinction is relevant :thinking: 
It is true that a block appear if you restart or rebuild
But you didn't mention here other source of apparition (maintenance / monitoring action described on the second half) or the possibility they exist, or the fact that these block likely come from user activity...
But maybe it is better to introduce block and after introduce non-user cause

Reviewer: Another kind of block exist : UNDEPLOY, when you stop your application
![image](https://github.com/user-attachments/assets/11572026-e0e0-4743-9e29-1fa19dac2197)
In that case there is no commit id, nor mean to redeploy this commit

Author: ```suggestion
Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application restarts or rebuilds, a new block appears on the timeline. Here's what you might see on the timeline:
```

> Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application restarts or rebuilds, a new block appears on the timeline. Here's what you might see on the timeline:

Author: ```suggestion
Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application redeploys, a new block appears on the timeline. Re-deployment triggers can be either:

 - An action performed by you or by a member of your organization
 - An action performed by the monitoring
 
 ## Re-deployment triggered by you
 
 ... or by a member of your organization.
 
 Here's what you might see on the timeline:
```

![Activity timeline](/images/doc/activity.png)

Each block indicates if the deployment was successful or unsuccessful, along with the following information:

- **Date**: when the activity happened and from where (`Console`, `Git`, `Github` or `Monitoring`)
- **Author**: which member of your organization triggered the activity
- **Commit ID**: on which commit the activity happened
- **Redeploy this commit**: the option to redeploy a previous commit (useful to quickly roll-back if needed)

## Re-deployments triggered by the monitoring

> If the monitoring triggers a redeployment, you'll see no author and the message explaining the reason, like `Monitoring/Unreachable`. The messages you might get are the following:

Reviewer: Maybe add a hint here or later on how to debug application ? like get a look on metrics in grafana :thinking:

Author: ```suggestion
If the monitoring triggers a re-deployment, you'll see no author and the message explaining the reason, like `Monitoring/Unreachable`. The messages you might get are the following:
```

---
type: docs
title: Activity
description: Applications activity lifecycle explained
tags:
- lifecycle
keywords:
- activity
- unreachable
- timeout
- diskfull
- scaling
- maintenance
type: docs
---

Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application redeploys, a new block appears on the timeline. Re-deployment triggers can be either:

 - An action performed by you or by a member of your organization

>  - An action performed by the monitoring

Author: ```suggestion
 - An action performed by Clever Cloud
```

 
 ## Re-deployment triggered by you
 
This happens when you or a member of your organization performs an action on the application, like:

- [Restart or rebuild](/doc/administrate/apps-management/#start-restart-and-stop) from the Console or [the CLI](/doc/cli/)

> - Push a new commit from Git or GitHub

Reviewer: ```suggestion
- Push a new commit from Git or GitHub
- Stop your application
```

 
 Here's what you might see on the timeline:

![Activity timeline](/images/doc/activity.png)

Each block indicates if the deployment was successful or unsuccessful, along with the following information:

- **Date**: when the activity happened and from where (`Console`, `Git`, `Github` or `Monitoring`)
- **Author**: which member of your organization triggered the activity
- **Commit ID**: on which commit the activity happened
- **Redeploy this commit**: the option to redeploy a previous commit (useful to quickly roll-back if needed)

> ## Re-deployments triggered by the monitoring

Author: ```suggestion
## Re-deployments triggered by Clever Cloud
```

> If the monitoring triggers a re-deployment, you'll see no author and the message explaining the reason, like `Monitoring/Unreachable`. The messages you might get are the following:

Author: ```suggestion
Clever Cloud might also trigger re-deployments, either when you're being assisted by the Support, or automatically by the monitoring system. If the monitoring triggers a re-deployment, you'll see no author and the message explaining the reason, like `Monitoring/Unreachable`. The messages you might get are the following:
```

---
type: docs
title: Activity
description: Applications activity lifecycle explained
tags:
- lifecycle
keywords:
- activity
- unreachable
- timeout
- diskfull
- scaling
- maintenance
type: docs
---

Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application redeploys, a new block appears on the timeline. Re-deployment triggers can be either:

 - An action performed by you or by a member of your organization
 - An action performed by Clever Cloud
 
 ## Re-deployment triggered by you
 
This happens when you or a member of your organization performs an action on the application, like:

- [Restart or rebuild](/doc/administrate/apps-management/#start-restart-and-stop) from the Console or [the CLI](/doc/cli/)
- Push a new commit from Git or GitHub

> - Stop your application

Author: ```suggestion
```
